packages/manager/apps/dedicated/client/app/ip/ip-range.service.js

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable class-methods-use-this */
 import forEach from 'lodash/forEach';
 import padStart from 'lodash/padStart';
 import times from 'lodash/times';
 
-angular.module('Module.ip.services').service('IpRange', function IpRange() {
-  function convertIpToBin(_ip) {
+export default class {
```

Comment:
- conversion to a `class`
- disabling need for `this` by `ESlint`, it would have too much impact
- nothing else has been done